src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java

```diff
@@ -44,7 +45,11 @@
                 flatMap(pprl -> pprl.getReads().stream().map(read -> read.getMappingQuality()).filter(mq -> mq != QualityUtils.MAPPING_QUALITY_UNAVAILABLE))
                 .collect(Collectors.toList());
         final double rms = MathUtils.rms(qualities);
-        return Collections.singletonMap(getKeyNames().get(0), String.format("%.2f", rms));
+        return Collections.singletonMap(getKeyNames().get(0), formatedValue(rms));
+    }
+
+    public static String formatedValue(double rms) {
+        return String.format("%.2f", rms);
```

Comment:
same reason - testing. I made both of them package-public now and added  `@VisibleForTesting`